src/test/user.js

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../index';
+
+chai.use(chaiHttp);
+chai.should();
+
+describe('users', () => {
+  it('User should be registered', (done) => {
+    const user = {
+      username: 'Kagabo',
+      email: 'kagabo@gmail.com',
+      password: '12345678',
+    };
+    chai.request(app)
+      .post('/api/users')
+      .send(user)
+      .end((req, res) => {
+        res.should.have.status(200);
+        res.body.should.be.an('object');
+        res.body.user.should.have.property('username');
+        res.body.user.should.have.property('email');
+        res.body.user.should.have.property('token');
+        done();
+      });
+  });
+
+  it('User should be not be registered', (done) => {
```

Comment:
Update this description: It should also contain the reason why the user should be registered e.g Existing user should not be registered again